proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -19,10 +19,22 @@ option go_package = "github.com/cosmos/cosmos-sdk/x/staking/types";
 // recent HistoricalInfo
 // (`n` is set by the staking module's `historical_entries` parameter).
 message HistoricalInfo {
+  option deprecated = true;
+
   tendermint.types.Header header = 1 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
   repeated Validator      valset = 2 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
 }
 
+// Historical contains a set of minimum values needed for evaluating historical validator sets and blocks.
+// It is stored as part of staking module's state, which persists the `n` most
+// recent HistoricalInfo
+// (`n` is set by the staking module's `historical_entries` parameter).
+message HistoricalRecord {
+  bytes                     apphash        = 1;
+  google.protobuf.Timestamp time           = 2 [(gogoproto.stdtime) = true];
+  bytes                     validator_hash = 3;
```

Comment:
Sorry, I know this is just merged, but should this field be `validators_hash` (plural). I can make a PR if you want :)

Reply:
true i can amend